clang/test/Modules/compare-file-size.py

```diff
@@ -0,0 +1,22 @@
+import argparse
+import os
+
+def get_file_size(file_path):
+    try:
+        return os.path.getsize(file_path)
+    except:
+        print(f"Unable to get file size of {file_path}")
+        return None
+
+def main():
```

Comment:
Please add some documentation about what this function does, and its usage. Other python files used in tests should give you an idea about what's appropriate.